### DISCUSSION TITLE
Add Code of Conduct

### DISCUSSION THREAD

**Discussion by imdevarsh** [CatalystDevOrg/Catalyst#24] [Proposals]
It will make the contributors not be mean, and enforce guidelines for the community.

**Comment by imdevarsh** [comment]
Nevermind

**Comment by imdevarsh** [comment]
Now that it's here,
> All community leaders are obligated to respect the privacy and security of the reporter of any incident.

But I thought the Discussions tab was available for everyone to see?

**Reply by jdev082** [reply]
In the Catalyst project, the CODE of Conduct is to be referred to, not enforced. As most of what is in this file me and the community cannot understand.

**Reply by jdev082** [reply]
@webdev03 the CoC (Code of Conduct) was generated by Github

**Reply by imdevarsh** [reply]
It's the Contributor Covenant. You put in the discussions tab thing.
